.travis.yml

```diff
@@ -20,6 +20,7 @@ env:
   - SCRIPT="./dev-docker ./manage.py test"
   - SCRIPT="./scripts/test-docker"
   - SCRIPT="./manage.py compilemessages" INSTALLPIP=yes
+  - SCRIPT="docker run -v $(PWD):/src --workdir /src fsfe/reuse reuse lint"
```

Comment:
I think you mean `squad` instead of `src`, which is where reuse will scan our source files.

Reply:
Ack, thanks @chaws